diff --git a/node_modules/@types/express-serve-static-core/index.d.ts b/node_modules/@types/express-serve-static-core/index.d.ts
index 4f1fc77..9676208 100644
--- a/node_modules/@types/express-serve-static-core/index.d.ts
+++ b/node_modules/@types/express-serve-static-core/index.d.ts
@@ -42,7 +42,7 @@ export interface Dictionary<T> {
 }
 
 export interface ParamsDictionary {
-    [key: string]: string | string[];
+    [key: string]: string;
     [key: number]: string;
 }
 export interface ParamsFlatDictionary {
